deepchecks/core/check_result.py

```diff
@@ -503,4 +503,4 @@ def __repr__(self):
         """Return string representation."""
         tb_str = traceback.format_exception(etype=type(self.exception), value=self.exception,
```

Comment:
don't you need to remove this var?

Reply:
Just checked how it looked, left it as is, `print(check_failure)` will still display the whole error trace